src/Tearing/TearingEngine.inl

```diff
@@ -484,6 +485,7 @@ inline bool TearingEngine<DataTypes>::computeEndPointsNeighboringTriangles(Coord
 template<class DataTypes>
 inline bool TearingEngine<DataTypes>::computeIntersectionNeighborTriangle(Coord normalizedFractureDirection,Coord Pa ,Coord& Pb, Real& t)
 {
+    SOFA_UNUSED(Pa);
```

Comment:
The 'Pa' parameter can be removed from this function's argument. Consequently, it should also be removed from the function 'computeEndPointsNeighboringTriangles' accordingly.

Reply:
ok, good to know, we will remove it in another PR